### DISCUSSION TITLE
Does Bright Data (ex-Luminati) work

### DISCUSSION THREAD

**Discussion by sdspieg** [scholarly-python-package/scholarly#443] [Q&A]
Has anybody tried this recently - using the pg.Luminati with Bright Data? Does it work?

**Comment by arunkannawadi** [comment]
Since they don't offer a free (even if limited) plan, I haven't been able to check this. For starters, you can check if `scholarly` communicates with BrightData by running

```python
python -m uniittest test_module.TestLuminati
```
